src/components/SearchInput/index.tsx

```diff
@@ -0,0 +1,157 @@
+import Image from "next/image";
+import React, { FC, useCallback, useEffect, useState } from "react";
+import { useRecoilState } from "recoil";
+import styled from "styled-components";
+
+import RecentKeywords from "./RecentKeywords";
+import SearchResults from "./SearchResults";
+
+import KeywordSearchResult from "types/KeywordSearchResult";
+import LatLng from "types/LatLng";
+import searchKeyword from "utils/apis/SearchApi";
+import { centerState } from "utils/states/naverMapState";
+
+const RECENT_KEYWORD_KEY = "recentKeywords";
+
+const SearchInput: FC = () => {
+  const [, setCenter] = useRecoilState(centerState);
```

Comment:
```suggestion
  const setCenter = useSetRecoilState(centerState);
```

useSetRecoilState 사용하면 setter만 가져올 수 있어서 요게 깔끔해 보이는데 어때??